atomic_reactor/inner.py

```diff
@@ -379,16 +379,6 @@ def build_docker_image(self):
             except Exception:
                 logger.exception("failed to save build duration")
 
-            self.build_logs = build_result.logs
-
-            self.build_failed = build_result.is_failed()
-
-            if build_result.is_failed():
-                # The docker build failed. Finish here, just run the
-                # exit plugins (from the 'finally:' block below).
-                self.plugins_errors['dockerbuild'] = ''
-                return build_result
```

Comment:
OK, I'm going to let this go for this PR since it's largely about getting unit tests in order, but I will remember it when we want to merge into master. :-)